pkg/stdio/console.go

```diff
@@ -87,19 +87,23 @@ func (t *terminalStdio) GetSecret(prompt string, minLen int) (password string, e
 // GetPassword prompts the user for a secret twice, and inputs must match.
 // Uses stdio.MinPasswordLen as the minimum input length
 func (t *terminalStdio) GetPassword(prompt string) (string, error) {
-	pass1, err := t.GetSecret(prompt, MinPasswordLen)
-	if err != nil {
-		return "", err
+	for {
+		pass1, err := t.GetSecret(prompt, MinPasswordLen)
```

Comment:
This call will already retry internally if the user inputs a value that's too short